### PR TITLE
Fix `RobertaPreLayerNorm` doctest

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Fix `RobertaPreLayerNorm` doctest. The doctest should have 0 failure against this commit 🔥

### CODE REVIEW

**Review by ydshieh — COMMENTED** [review]
A few comments.

File: src/transformers/models/roberta_prelayernorm/modeling_roberta_prelayernorm.py [hunk 1]
```diff
@@ -1075,6 +1075,9 @@ def set_output_embeddings(self, new_embeddings):
         checkpoint=_CHECKPOINT_FOR_DOC,
         output_type=MaskedLMOutput,
         config_class=_CONFIG_FOR_DOC,
+        mask="<mask>",
```
[the remark this answers is not in the document]
Comment (ArthurZucker) [reply src/transformers/models/roberta_prelayernorm/modeling_roberta_prelayernorm.py:1078]: I think this was removed in order not evaluate this, but if this passes cool

File: src/transformers/models/roberta_prelayernorm/modeling_tf_roberta_prelayernorm.py [hunk 2]
```diff
@@ -1095,7 +1095,7 @@ def get_prefix_bias_name(self):
         config_class=_CONFIG_FOR_DOC,
         mask="<mask>",
         expected_output="' Paris'",
-        expected_loss=0.1,
+        expected_loss=0.69,
```
[the remark this answers is not in the document]
Comment (ArthurZucker) [reply src/transformers/models/roberta_prelayernorm/modeling_tf_roberta_prelayernorm.py:1098]: Same here, if the checkpoint is not random, cool

File: src/transformers/models/roberta_prelayernorm/modeling_tf_roberta_prelayernorm.py [hunk 3]
```diff
@@ -1354,8 +1354,6 @@ def __init__(self, config, *inputs, **kwargs):
         checkpoint=_CHECKPOINT_FOR_DOC,
         output_type=TFSequenceClassifierOutput,
         config_class=_CONFIG_FOR_DOC,
-        expected_output="'optimism'",
```
[the remark this answers is not in the document]
Comment (ArthurZucker) [reply src/transformers/models/roberta_prelayernorm/modeling_tf_roberta_prelayernorm.py:1357]: NIce, I thought I had removed this but no!

**Review by ArthurZucker — APPROVED** [review]
Thanks a lot for fixing this!

**Review by sgugger — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**